cEP-0016.md

```diff
@@ -0,0 +1,178 @@
+Git Commit Content Inspection
+========================
+
+|Metadata      |                                            |
+|--------------|--------------------------------------------|
+| cEP          | 16                                         |
+| Version      | 1.0                                        |
+| Title        | Git commit inspection                      |
+| Authors      | Kriti Rohillla <kritirohilla567@gmail.com> |
+| Status       | Implementation due                         |
+| Type         | Process                                    |
+
+# Abstract
```

Comment:
I think we can change this "Header 1" title to a "Heading 2" title (`## Abstract`)
and also change the "Header 1" which are below this line